glTFLoader/Interface.cs

```diff
@@ -151,6 +175,39 @@ private static void ReadBinaryHeader(BinaryReader binaryReader)
             }
         }
 
+        public static Stream OpenImageFile(this Gltf model, string gltfFilePath, int imageIndex)
+        {
+            var image = model.Images[imageIndex];
+
+            if (image.BufferView.HasValue)
+            {
+                var bufferView = model.BufferViews[image.BufferView.Value];
+
+                var bufferBytes = model.LoadBinaryBuffer(gltfFilePath, bufferView.Buffer);
+
+                return new MemoryStream(bufferBytes, bufferView.ByteOffset, bufferView.ByteLength);
+            }
+            
+            if (image.Uri.StartsWith("data:image/"))
+            {
+                string content = null;
+
+                if (image.Uri.StartsWith(EMBEDDEDPNG)) content = image.Uri.Substring(EMBEDDEDPNG.Length + 1);

```

Comment:
glTF only support PNG and JPG. None of the other formats are supported, so you don't need the other mime types.

https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#file-extensions-and-mime-types